### PR TITLE
feat: document network namespace connectivity collector

### PR DESCRIPTION
**Pull request by ricardomaraschini**
documents the network namespace connectivity collector and analyzer.

### CODE REVIEW

File: docs/source/host-collect-analyze/networkNamespaceConnectivity.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+The `NetworkNamespaceConnectivity` analyzer supports two outcomes, one for `pass` and one for `fail`. An example is provided below.
+
+## Message Templating
+
+To make the outcome message more informative, you can include certain values gathered by the `NetworkNamespaceConnectivity` collector as templates. The templates are enclosed in double curly braces with a dot separator. The following templates are available:
+
+| Template | Description |
+|----|----|
+|`{{ .ErrorMessage }}` | Show all error messages found during the collection |
```
Comment (banjoh) [inline docs/source/host-collect-analyze/networkNamespaceConnectivity.md:101]: It does not look like `.ErrorMessage` field exists

https://github.com/replicatedhq/troubleshoot/blob/e272683bced0b611553c328679bd1f5f7dd9228f/pkg/collect/host_network_namespace_connectivity.go#L20

**Review by banjoh — CHANGES_REQUESTED** [review]

File: docs/source/host-collect-analyze/networkNamespaceConnectivity.md (hunk 1 above)
Comment (ricardomaraschini) [reply docs/source/host-collect-analyze/networkNamespaceConnectivity.md:101]: https://github.com/replicatedhq/troubleshoot/blob/main/pkg/collect/host_network_namespace_connectivity.go#L29

File: docs/source/host-collect-analyze/networkNamespaceConnectivity.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+The CIDR to be used on the second network namespace.
+
+##### `port` (Optional)
+
+The port to use for the UDP and TCP connections. Defaults to `8080`.
+
+##### `timeout` (Optional)
+
+The time to wait for the UDP and TCP connections to be established. This parameter is expressed with a string following the [Go duration format](https://pkg.go.dev/time#ParseDuration).
```
[the remark this answers is not in the document]
Comment (ricardomaraschini) [reply docs/source/host-collect-analyze/networkNamespaceConnectivity.md]: done.

File: docs/source/host-collect-analyze/networkNamespaceConnectivity.md (hunk 1 above)
Comment (banjoh) [reply docs/source/host-collect-analyze/networkNamespaceConnectivity.md:101]: Aaah. :D

**Pull request merged**